taskcluster/client.py

```diff
@@ -496,6 +496,10 @@ def _makeHttpRequest(self, method, route, payload):
             if status == 204:
                 return None
 
+            # Handle content length 0 (empty server responses)
+            if header("Content-Length: ".filesize(response)) == 0:
```

Comment:
Hmm, I'm not sure if this is right.  I'm not sure what's intended here.  The `header` is actually a python dictionary, so you'd either need to use subscripting, or the `.get()`/`.has()` accessors.

I'm also not sure about the `.filesize` method.  I don't see that as a method on the string type.

```
Python 2.7.10 (default, Jul 15 2017, 17:16:57)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> "a-string".filesize
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'str' object has no attribute 'filesize'
```

Reply:
Also, `header()` is not a defined function.
```
>>> header()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'header' is not defined
>>> 
```